daphne/cli.py

```diff
@@ -146,6 +168,38 @@ def entrypoint(cls):
         """
         cls().run(sys.argv[1:])
 
+    def _check_proxy_headers_passed(self, argument: str, args: Namespace):
+        """Raise if the `--proxy-headers` weren't specified."""
+        if args.proxy_headers:
+            return
+        raise ArgumentError(
+            argument=argument,
+            message="--proxy-headers has to be passed for this parameter.")
+
+    def _get_forwarded_host(self, args: Namespace) -> str_or_none:
+        """
+        Return the default host header from which the remote hostname/ip
+        will be extracted.
+        """
+        if args.proxy_headers_host:
+            self._check_proxy_headers_passed(
+                argument=self._arg_proxy_host, args=args)
```

Comment:
Don't need to pass the string like this, just redeclare it.

Reply:
`self._arg_proxy_host` is not a string, but a parameter that's used for formatting the error messages. If you look at what return value `add_argument` returns, you'll see.

Reply:
Oh, I know what it is. I'm saying you can just use a string here instead.